### ISSUE TITLE
Root package.json and package-lock.json out of sync

### ISSUE THREAD

**Issue by or-schneider** [sharkio-dev/sharkio#174]
**tldr**
Needs to push an updated package-lock.json file

**Describe the bug**
npm install causes changes in package-lock

**To Reproduce**
clone project, npm install
package-lock.json will be modified

**Expected behavior**
package-lock should stay the same after npm install

**Additional context**
Running npm ci (clean-install) gives a nice descriptive error
"`npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing."

At the very least the dev dependency 'pretty-quick' in package.json is missing in package-lock.json

**Comment by or-schneider** [comment]
@idodav You can assign this to me 
In case it'll matter my npm version is 9.8.1 which is currently the latest version

**Comment by idodav** [comment]
@or-schneider assigned to you

**Comment by or-schneider** [comment]
@idodav I see yarn.lock was added yesterday in d4236e0fc451e8ae9139b3fb0bbfba59c6b2ff5c
Which lock should be used going forward, yarn.lock or package-lock.json?

**Comment by nkotzrenko** [comment]
Fix referenced in https://github.com/sharkio-dev/sharkio/pull/181

**Comment by idodav** [comment]
> lock

package-lock should be used!
